srcpkgs/lastpass-cli/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'lastpass-cli'
 pkgname=lastpass-cli
-version=1.3.2
-revision=2
+version=1.3.3
+revision=3
```

Comment:
The revision should be reset to 1 when the package is updated to a new version, not increased.